ulp-observability-front/src/main/front/src/app/app.module.ts

```diff
@@ -53,9 +55,23 @@ export class MaterialModule {};
     NgChartsModule,
     BrowserAnimationsModule,
     MaterialModule,
-    FlexLayoutModule
+    FlexLayoutModule,
+    TranslateModule.forRoot({
+      defaultLanguage: 'fr',
```

Comment:
defaultLanguage should be en

Reply:
Maybe you need to add something so that the language is taken from server side ? 
There is a config url that may need to be enriched get the language from  JMeterUtils.getLocale() on the client side. And you would then adapt code to take it into account on client side